src/Sulu/Bundle/WebsiteBundle/Resources/views/Extension/seo.html.twig

```diff
@@ -6,6 +6,19 @@
  # localizations array
 -#}
 
+{% if localizationsa is not defined %}
```

Comment:
`localizationsa` should be `localizations` you can test it by adding `only` keyword behind the include of the in the base.html.twig e.g.:

```
{% include "@SuluWebsite/Extension/seo.html.twig" with {
            "seo": extension.seo|default([]),
            "content": content|default([]),
            "urls": localizations|default([]),
            "shadowBaseLocale": shadowBaseLocale|default(),
            "defaultLocale": app.request.locale
} only %}
```